src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -4514,7 +4514,7 @@ private MemberAst ClassMemberRule(string className, out List<Ast> astsOnError)
                 }
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By CustomPipeName.Can enter, exit, and re-enter using CustomPipeName
```powershell
Expected $true, because The script was able to re-enter another process and grab the pipe of 'akl3gcul.0uh'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 170
170:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pipe of '$pipeName'."
```